Agents/GFAAgent/stack-manager-input-config/data.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "Augmented Singapore",
+  "stack": "http://localhost:3838",
+  "sources": [
+    {
+      "id": "buildings",
+      "type": "vector",
+      "tiles": [
+        "http://localhost:3838/geoserver/twa/wms?service=WMS&version=1.1.0&request=GetMap&layers=twa:mainland&bbox={bbox-epsg-3857}&width=256&height=256&srs=EPSG:3857&format=application/vnd.mapbox-vector-tile"
+      ]
```

Comment:
Please remove this data.json as the agent should not be setting up the visualisation. And the data.json is specific to Singapore so it will vary